client_plugin/drivers/shared/shared_driver.go

```diff
@@ -23,24 +23,48 @@ package shared
 ///
 
 import (
+	"context"
+	"fmt"
 	log "github.com/Sirupsen/logrus"
 	dockerClient "github.com/docker/engine-api/client"
+	dockerTypes "github.com/docker/engine-api/types"
 	"github.com/docker/go-plugins-helpers/volume"
 	"github.com/vmware/docker-volume-vsphere/client_plugin/drivers/utils"
 	"github.com/vmware/docker-volume-vsphere/client_plugin/utils/config"
 	"github.com/vmware/docker-volume-vsphere/client_plugin/utils/refcount"
+	"os"
 )
 
 const (
-	version       = "vSphere Shared Volume Driver v0.2"
-	apiVersion    = "v1.24" // docker engine 1.24 and above support this api version
-	dockerUSocket = "unix:///var/run/docker.sock"
+	version              = "vSphere Shared Volume Driver v0.2"
+	apiVersion           = "v1.24" // docker engine 1.24 and above support this api version
+	dockerUSocket        = "unix:///var/run/docker.sock"
+	tradVolumePrefix     = "InternalTrad"
+	volStateCreating     = "Creating"
+	volStateReady        = "Ready"
+	volStateMounted      = "Mounted"
+	volStateIntermediate = "Intermediate"
+	sambaImageName       = "dperson/samba" // TODO Replace with our own samba later
```

Comment:
I suggest to replace it now. Just push manually to cnastorage/shared-vmdk-samba or something like that. 
Would be easier that to track it